tests/conftest.py

```diff
@@ -57,9 +57,3 @@ def _db(app):
     Flask-SQLAlchemy database connection.
     """
     return DB
-
-
-@pytest.fixture
-def clear_cache(app):
```

Comment:
@ammaratef45 I dont see a way to delete it. We definitely should clear cache before every test

Reply:
@vladarefiev that was the puzzle #305 
I may clear the cache in the app fixture
What do you think?

Reply:
@ammaratef45 I think no need to make one fixture more complexity.
I prefer leave this one with `autouse=True` may be